plugins/woocommerce-blocks/assets/js/blocks/collection-filters/edit.tsx

```diff
@@ -1,35 +1,64 @@
 /**
  * External dependencies
  */
-import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
+import { InnerBlocks } from '@wordpress/block-editor';
```

Comment:
We need to add the import for `useBlockProps` back, I'm sorry I didn't add this suggestion before.
```suggestion
import { useBlockProps, InnerBlocks } from '@wordpress/block-editor';
```

Reply:
Whoops no probs. I'll do this manually in my editor.

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/43216/commits/27e44602e6931ca735c243142b2b9cae483f5295